dask_ec2/formulas/salt/dask/distributed/init.sls

```diff
@@ -4,24 +4,21 @@ include:
   - conda
   - system.base
 
-dask-install:
-  pip.installed:
-    - name: dask
-    - bin_env: {{ install_prefix }}/bin/pip
+bokeh-install:
+  cmd.run:
+    - name: {{ install_prefix }}/bin/conda install bokeh -y -q
     - require:
       - sls: conda
 
-bokeh-install:
+dask-install:
   cmd.run:
-    - name: {{ install_prefix }}/bin/conda install bokeh -y -q
+    - name: {{ install_prefix }}/bin/conda install dask -y -q -c conda-forge
     - require:
       - sls: conda
 
 distributed-install:
-  pip.installed:
-    - name: distributed
-    - upgrade: true
-    - bin_env: {{ install_prefix }}/bin/pip
+  cmd.run:
+    - name: {{ install_prefix }}/bin/conda install distributed -y -q -c conda-forge
```

Comment:
I don't think there is any reason to split up the dask and distributed installs.  They can both be installed at the same time and it would reduce the overhead of conda being called twice